modules/core/src/main/scala/org/scalasteward/core/repoconfig/RepoConfigAlg.scala

```diff
@@ -86,8 +86,10 @@ object RepoConfigAlg {
   private def activeConfigFile[F[_]](
       repoDir: File
   )(implicit fileAlg: FileAlg[F], logger: Logger[F], F: Monad[F]): F[Option[File]] = {
-    val configFileCandidates: F[List[File]] = repoConfigFileSearchPath
-      .map(_ :+ repoConfigBasename)
+    val configFileCandidates: F[List[File]] = (repoConfigFileSearchPath
+      .map(_ :+ repoConfigBasename) ++
+      repoConfigFileSearchPath
+        .map(_ :+ repoConfigBasename.substring(1)))
```

Comment:
The fix is here: The `.substring(1)` removes the leading dot from the filename.
As you can see I just append the possible non-hidden config file locations to the end of the list, which later will be searched for the first existing config file.

Reply:
Would it make sense to change the repoConfigBasename to be without the `.` instead, and then add the dot here? substring use here adds a bit of a mental overhead.

PS: not associated with the project, but got a notification because of my PR and had to look at the code! Feel free to ignore :)

Reply:
The `repoConfigBasename` variable is publicly accessible therefore I don't want to change it's value for backwards compatibility.

Reply:
makes sense!

Reply:
In line 46 of this file it maybe good to use the actual filename (`activeConfigFile`) instead of `repoConfigBasename`

Reply:
@mzuehlke Done and pushed, thanks!